modular_skyrat/modules/GAGS/json_configs/respirator/respirator.json

```
@@ -0,0 +1,10 @@
+{
+	"respirator": [
+		{
+			"type": "icon_state",
+			"icon_state": "respirator_obj",
+			"blend_mode": "overlay",
+			"color_ids": [ 1 ]
+		}
+	]
+}
```

```suggestion
{
	"respirator": [
		{
			"type": "icon_state",
			"icon_state": "respirator_obj",
			"blend_mode": "overlay",
			"color_ids": [ 1 ]
		}
	],
	"respirator_worn": [
		{
			"type": "icon_state",
			"icon_state": "respirator",
			"blend_mode": "overlay",
			"color_ids": [ 1 ]
		}
	]
}
```

Rather than have several .jsons per item, what we can also do is have them as separate icon_states in the same file (see suggestion)
Alternatively, we could do what I implemented on a few items on TG: have the item/worn config link to a separate .dmi with the same icon names, letting it inherit the same .json and effectively use the same one - that one's better for when the items are kept in universal files, like the sorting I've been doing with uniforms, so it doesn't quite apply here as these have their own .dmis

None of this is enforced nor agreed on, thus not at all required to change, but it may be nicer either on the code or just on the psyche to not add several new files for greyscale